services/transaction_record.proto

```diff
@@ -120,6 +120,9 @@ message TransactionRecord {
     /**
      * In the record of an internal CryptoCreate transaction triggered by a user 
      * transaction with a (previously unused) alias, the new account's alias. 
+     * 
+     * In the record of an internal ContractCreate transaction triggered by a user
+     * transaction, the new contract's Solidity address.
      */
      bytes alias = 16;
```

Comment:
I think it might make more sense to populate a `solidity_address` on the `ContractFunctionResult` instead of overloading alias. On the result is useful for the mirror node so that the mapping between contract ID and solidity address is explicit and we don't have to assume it's the shard/realm/num format for some and not others.

Reply:
Added a `solidity_address` field. 👍

Reply:
Wouldn't it belong on the `ContractFunctionResult`?

Reply:
Good point, moved to `ContractFunctionResult`.